### DISCUSSION TITLE
Python sdf module manual

### DISCUSSION THREAD

**Discussion by zawang727** [epochpic/epoch#299] [Q&A] [answered]
I plan to use python sdf module to visualize sdf data.
However, is there any manual or description for the python sdf module.
I can not get the attribute of the contained data.
Thanks for your help!

**Comment by TomGoffrey** [comment, marked as answer]
Perhaps this is helpful:

https://epochpic.github.io/documentation/visualising_output/python.html

**Reply by zawang727** [reply]
Thanks for your help!
That is what I need.

**Reply by erny123** [reply]
On this topic, have you guys thought about adding your SDF library to a standard repository such as Anaconda? 
It would make it much easier to use on general computers.

**Reply by zawang727** [reply]
I am agree with erny. If sdf_helper can be installed on Anaconda. The postprocessing process will be much easier. However, I don't know how to install sdf_helper on Windows Anaconda. I think it is hard because I have to compile the linux based code on Windows.
